task/udpecho/src/main.rs

```diff
@@ -40,8 +40,10 @@ fn main() -> ! {
                         Err(NetError::NotYours) => panic!(),
                         Err(NetError::InvalidVLan) => panic!(),
                         Err(NetError::Other) => panic!(),
-                        // `send_packet()` can't return QueueEmpty
+                        // `send_packet()` can't return these errors
```

Comment:
As these diverge more, it becomes more likely that we should separate these error types. There's no particular reason why every operation on a server needs to return the same error type, and in these cases it seems like we're approaching having disjoint sets of "MAC related errors" vs "PHY related errors."